pkg/features/kube_features.go

```diff
@@ -1440,6 +1440,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 
 	MaxUnavailableStatefulSet: {
 		{Version: version.MustParse("1.24"), Default: false, PreRelease: featuregate.Alpha},
+		{Version: version.MustParse("1.35"), Default: true, PreRelease: featuregate.Beta},
```

Comment:
Since you're touching this file already, can you add the kep link:
```
	// kep: https://kep.k8s.io/961
```
here: https://github.com/kubernetes/kubernetes/blob/11ade2f7dd264c2f52a4a1342458abbbaa3cb2b1/pkg/features/kube_features.go#L577-L580 so that it's easier to find, and matches what we're doing for other feature gates.